platform-sdk/swirlds-state-api/src/testFixtures/java/module-info.java

```diff
@@ -0,0 +1,15 @@
+open module com.swirlds.state.api.test.fixtures {
+    requires transitive com.hedera.node.hapi;
+    requires transitive com.swirlds.common;
+    requires transitive com.swirlds.merkle;
+    requires transitive com.swirlds.state.api;
+    requires transitive com.swirlds.virtualmap;
+    requires transitive com.hedera.pbj.runtime;
+    requires transitive org.junit.jupiter.params;
```

Comment:
why is that one transitive?

Reply:
I'm getting this error without `tranisitive`:
```
MerkleTestBase.java:362: warning: [exports] interface Arguments in module org.junit.jupiter.params is
not indirectly exported using 'requires transitive'
    public static Stream<Arguments> illegalServiceNames() {
```